RocketRL/python/envs/RocketRL/RocketEnv_2T.py

```diff
@@ -0,0 +1,421 @@
+"""This code is adapted from rllib's example of a custom env"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import gym
+from gym import spaces
+from gym.envs.registration import EnvSpec
+import random
+import pandas as pd
+import pickle
+
+
+import ray
+from ray.tune import run_experiments
+from ray.tune.registry import register_env
+
+import os
+import yaml
+
+# ##TO DO: get cfg in here more elegantly
+#set config path
+
+#TO DO: change this to get to this dir from the ray dir: 
+
+CWD_PATH = '/Users/ninalopatina/gitrepos/Rocket_RL/'
```

Comment:
Can you remove this hardcoded dependency? Maybe `os.getcwd()` will help.

Reply:
This gets complicated because this env is run from 2 places, the gitrepo, and also during the rollout w/the command

python ray/python/ray/rllib/rollout.py ~/ray_results/RocketRL/PPO_TwoTemp_0_2018-06-28_09-56-296vlgo7qq/checkpoint-76 --run PPO --env TwoTemp-v0 --steps 50000

If I have the user run the main script from their home directory, it could access both my package and the ray package. So I can change this and then change all my path definitions to include the path to the git repo or rllib, depending on which needs to be accessed